CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ A [separate changelog is kept for rand_core](rand_core/CHANGELOG.md).
 
 You may also find the [Upgrade Guide](https://rust-random.github.io/book/update.html) useful.
 
+## [Unreleased]
+### Removals
```

Comment:
I don't think it's correct to use "removals" for these change. We either need a separate section "deprecations" or put it into "changes".

BTW FYI the keepchangelog recommends to use "added", "changed", "removed", etc. for section names.

Reply:
A deprecation is a non-breaking removal... we already have both `Removed` and `Deprecated` titles. keepachangelog actually recommends both.

Reply:
I meant that the `rand` changelog [uses](https://github.com/rust-random/rand/blob/fdaf424ff46ab7124787ea66f03484dc58c53b9e/CHANGELOG.md#091---2025-04-17) "Additions", "Changes", "Fixes", etc. instead of the recommended "Added", "Changed", "Fixed", etc. But I guess it's better to discuss changelog format in a separate issue.